docs/displaying-ads-hook.mdx

```diff
@@ -84,7 +84,7 @@ Return values of the hook are:
 
 | Name           | Type                               | Description                                                                                                        |
 | :------------- | :--------------------------------- | :----------------------------------------------------------------------------------------------------------------- |
-| isLoaded       | boolean                            | Whether the ad is loaded and ready to to be shown to the user.                                                     |
+| isLoaded       | boolean                            | Whether the ad is loaded and ready to to be shown to the user. Automatically set to false when the ad is opened.   |
```

Comment:
I think we can leave the original text untouched or change to something like "Automatically set to `false` when the ad was shown.".